authentication-service/src/main/kotlin/com/saveourtool/save/authservice/repository/AuthenticationUserRepository.kt

```diff
@@ -9,7 +9,7 @@ import org.springframework.stereotype.Component
  * Repository for [com.saveourtool.save.entities.User]
  */
 @Component
-class SandboxUserRepository(
+class AuthenticationUserRepository(
```

Comment:
This class relies on querying a particular schema (`save_cloud`, while previously this component was operating on `save_cloud_sandbox`), is this OK for a separate service?